CHANGELOG_PENDING.md

```diff
@@ -17,6 +17,11 @@ Friendly reminder, we have a [bug bounty program](https://hackerone.com/tendermi
 
 - Go API
 
+  - [privval] [\#4744](https://github.com/tendermint/tendermint/pull/4744) Remove deprecated `OldFilePV` (@melekes)
+
+- Blockchain Protocol
+
+>>>>>>> f31f4327b... privval: remove deprecated `OldFilePV`
```

Comment:
Conflict marker. The above Blockchain Protocol should probably not be here either.